src/validations.test.ts

```diff
@@ -1,18 +1,24 @@
 import {canBeConvertedToPDF} from './validations';
 
-describe('canBeConvertedToPDF', () => {
-  it.each`
-    filename        | expected
-    ${'image.jpg'}  | ${false}
-    ${'image.mp4'}  | ${false}
-    ${'image.mp3'}  | ${false}
-    ${'image.wav'}  | ${false}
-    ${'image.chm'}  | ${false}
-    ${'image.gdoc'} | ${false}
-    ${'image.dwg'}  | ${false}
-    ${'image.docx'} | ${true}
-    ${'image.pdf'}  | ${true}
-  `('should return $expected for filename $filename', ({filename, expected}) => {
-    expect(canBeConvertedToPDF(filename)).toEqual(expected);
-  });
+it.each`
+  filename
+  ${'image.docx'}
+  ${'image.pdf'}
+`('should return true for supported filename $filename', ({filename}) => {
```

Comment:
Cool idea with splitting test cases into supported/unsupported, it became clearer 👍